docs/_docs/01_getting-started/quick-start.md

```diff
@@ -131,8 +135,11 @@ func TestWebServer(t *testing.T) {
 }
 ```
 
-The code above does all the steps we mentioned above, including running `terraform init`, `terraform apply`, making HTTP requests to the web server (retrying up to 15 times with 5 seconds between retries), and running `terraform destroy` (using [`defer`](https://blog.golang.org/defer-panic-and-recover) to run it at the end of the test, whether the test succeeds or fails). If you put this code in a file called `web_server_test.go`, you can run it by executing `go test`, and you’ll see output that looks like this (truncated for readability):
+The UnitTest function above does all the steps we mentioned above, including running `terraform init`, `terraform apply`, calling your validator to make HTTP requests to the web server (retrying up to 15 times with 5 seconds between retries), and running `terraform destroy` (using [`defer`](https://blog.golang.org/defer-panic-and-recover) to run it at the end of the test, whether the test succeeds or fails). You can easily add more validations by providing extra validation functions to the Validators list.

```

Comment:
Does the unit test run `apply` or `plan`? If it runs `apply`, what's the difference with integration test then?